src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequestMerge.java

```diff
@@ -37,11 +37,11 @@
 
     private transient PrintStream logger;
 
-    private final Boolean onlyAdminsMerge;
-    private final Boolean disallowOwnCode;
+    private final boolean onlyAdminsMerge;
```

Comment:
In my experience it is preferable to us the object here instead of a primitive incase it is null when the get method is called.  This is possible because Jenkins doesn't use the constructor to recreate objects when it restarts.